src/app.module.ts

```diff
@@ -1,9 +1,24 @@
 import { Module } from '@nestjs/common';
 import { AppController } from './app.controller';
 import { AppService } from './app.service';
+import { AuthModule } from './auth/auth.module';
+import { SequelizeModule } from '@nestjs/sequelize';
+import { UserModel } from './models/user.model';
+import { UserModule } from './user/user.module';
 
 @Module({
-  imports: [],
+  imports: [AuthModule, SequelizeModule.forRoot({
+    dialect: 'postgres',
+    host: 'localhost',
+    port: 5432,
+    username: 'postgres',
```

Comment:
Any configuration in the project should be got from the .env file, because it depends on the environment changing the values from development to staging to production environment, so keep them in the .env file, search more about it